docs/antrea-network-policy.md

```diff
@@ -1288,14 +1293,54 @@ spec:
 
 In this example, the policy will be applied to all Pods whose ServiceAccount is `sa-1` of `ns-1`.
 Let's call those Pods "appliedToPods".
-The egress `to` section will select all Pods whose ServiceAccount is in `ns-2` namespace and name as `sa-2`.
+The egress `to` section will select all Pods whose ServiceAccount is in `ns-2` Namespace and name as `sa-2`.
 Let's call those Pods "egressPods".
 After this policy is applied, traffic from "appliedToPods" to "egressPods" will be dropped.
 
 Note: Antrea will use a reserved label key for internal processing `serviceAccount`.
 The reserved label looks like: `internal.antrea.io/service-account:[ServiceAccountName]`. Users should avoid using
 this label key in any entities no matter if a policy with `serviceAccount` is applied in the cluster.
 
+### Apply to NodePort Service
+
+Antrea ClusterNetworkPolicy features a `service` field in `appliedTo` field to enforce the ACNP rules on the
+traffic from external clients to a NodePort Service.
+
+`service` uses `namespace` and `name` to select the Service with a specific name under a specific Namespace;
+only a NodePort Service can be referred by `service` field.
+
+There are a few **restrictions** on configuring a policy/rule that applies to NodePort Services:
```

Comment:
did we mention it can only work with kube-proxy disabled anywhere?

Reply:
Added in doc. Thanks!